companion/src/firmwares/boards.cpp

```diff
@@ -486,6 +486,9 @@ int Boards::getCapability(Board::Type board, Board::Capability capability)
     case NumFunctionSwitches:
       return IS_JUMPER_TPRO(board) ? 6 : 0;
 
+    case HasInternalModuleSupport:
+      return (IS_STM32(board) && !IS_TARANIS_SMALL(board));
```

Comment:
Should be `!IS_TARANIS_X9()`. `IS_TARANIS_SMALL()` has all the 128x64 which do have support for configurable internal module.